src/main/java/com/slack/eithernet/ApiResult.kt

```diff
@@ -308,6 +308,15 @@ public object ApiResultCallAdapterFactory : CallAdapter.Factory() {
                   val withTag =
                     when (val result = response.body()) {
                       is Success -> result.withTags(result.tags + tags)
+                      null -> {
+                        val successType = apiResultType.actualTypeArguments[0]
+                        val responseCode = response.code()
+                        if ((responseCode == 204 || responseCode == 205) && successType == Unit::class.java) {
+                          ApiResult.success(Unit).withTags(emptyMap<KClass<*>, Any>() + tags)
```

Comment:
Is the emptyMap necessary?

```suggestion
                          ApiResult.success(Unit).withTags(tags)
```

Reply:
Yes, the suggested code doesn't work. it fails because of 
```
Required:
Map<KClass<*>, Any>
Found:
Map<KClass<Response>, Response!>
```

Reply:
Actually found out a more elegant solution https://github.com/slackhq/EitherNet/pull/63/commits/30e8ebc041d9f327f44623f1af1eed9cc7bdc0cf